Cargo.toml

```diff
@@ -33,8 +33,8 @@ zeroize = { version = "1.5.5", default-features = false, features = [ "zeroize_d
 dotenv = { version =  "0.15.0", default-features = false }
 
 [features]
-# only default for now, should later be replaced by stronghold since stronghold is more secure
-default = [ "mnemonic", "storage", "stronghold" ]
+# mnemonic & storage should later be superseded by stronghold since stronghold is more secure
+default = [ "mnemonic", "events", "storage", "stronghold", "message_interface" ]
```

Comment:
Hmm, can we not put the features when using the wallet in the bindings ? Not sure these should be default tbh